### ISSUE TITLE
Destructive commit: string does not implement fmt.Stringer (missing String method)

### ISSUE THREAD

**Issue by iTanken** [golang-sql/sqlexp#2]
d42ba59f5c11011b1df5abc536ee75821c22bdac This is a destructive commit, after update <https://github.com/denisenkom/go-mssqldb/issues/745> compilation failed.

```shell
# github.com/denisenkom/go-mssqldb
vendor\github.com\denisenkom\go-mssqldb\token.go:790:70: cannot use info.Message (type string) as type fmt.Stringer in field value:
	string does not implement fmt.Stringer (missing String method)
```

**Comment by issueye** [comment]
+1

**Comment by ATAN13** [comment]
+1 TKyyds

**Comment by dsuhinin** [comment]
+1. Could you please start using versions to do not break related packages. thank you!

**Comment by syuichi-tsuji** [comment]
so am I. I got the same error.

**Comment by ai2hive** [comment]
+1

**Comment by iTanken** [comment]
Fixed #2 <https://github.com/denisenkom/go-mssqldb/commit/1598eaf61e43a1bb1b66231abe49f8613b544d64>
